lib/cadet_web/controllers/answer_controller.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CadetWeb.AnswerController do
+  use CadetWeb, :controller
+
+  use PhoenixSwagger
+
+  swagger_path :submit do
+    post("/assessments/question/{questionId}/submit")
+
+    summary("Submit an answer to a question")
+
```

Comment:
I think you should add `description("For MCQ, answer contains choice_id. For programming question, this is a string containing the code.")`